app/controllers/users_controller.rb

```diff
@@ -43,23 +43,21 @@ def index
   def show
     @user = User.find(params[:id])
     @slides = user_slide_with_paginate(params[:id])
-    # if params[:sort_by] == 'popularity'
-      # @slides = Slide.published.popular
-    # else
-      # @slides = Slide.published.latest
-    # end
-    # @slides = @slides.owner(params[:id]).
-            # includes(:user).
-            # paginate(page: params[:page], per_page: 30)
     @tags = Slide.published.
             owner(params[:id]).
             tag_counts_on(:tags).order('count DESC')
     respond_to do |format|
-      format.html
-      format.rss
+      format.html {}
+      format.rss {}
     end
   end
 
+  def embedded
+    @slides = user_slide_with_paginate(params[:id], 5)
+    s = render_to_string layout: 'plain', collection: @slide
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 's' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)